contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -0,0 +1,81 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import LoadCapture
+from test_cases.main_window import VerifyModalWindowExistsAndClickOkay
+from test_cases.live_tab import AddIterator, VerifyFunctionCallCount
+from test_cases.capture_window import CheckTimers, CollapseTrack, ExpandTrack, FilterTracks, CheckThreadStates, \
+    ToggleCollapsedStateOfAllTracks
+
+"""Verify loading a capture in Orbit using pywinauto.
+
+There is no prerequisites before this script is run.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test it needs
+to by run from 64 bit python.
+
+This automation script covers a basic workflow:
+ - load an old, unsupported capture
+ - load a supported capture
+ - verify that the scheduler track is present and contains timers
+ - verify that the frame track is present and contains timers
+ - verify that the tracks from manual instrumentation are present
+ - verify that the memory tracks are present
+ - verify that an iterator can be added to "TestFunc2"
+ - verify that "TestFunc2" was called exactly 1257 times
+"""
+
+
+def main(argv):
+    test_cases = [
+        LoadCapture(relative_capture_file_path="automation_tests\\test_data\\OrbitTest_1-64.orbit"),
+        VerifyModalWindowExistsAndClickOkay(window_name="Error while loading capture"),
+        LoadCapture(relative_capture_file_path="automation_tests\\test_data\\OrbitTest_1-72.orbit"),
```

Comment:
I think you haven't added the files to the commit.

Reply:
Yeah, good point. I stored them in the wrong "automation_tests" directory....

Reply:
Didn't we want to load them from GCS instead of submitting them to GitHub?

Reply:
Not for all test captures. This allows us to update this test with new features/versions, without requiring to change the compatibility settings in the internal infra. So it would simply run on the capture that is bundled with the python script.

Reply:
Ok, sounds reasonable. The main downside is that we can't test on larger captures, but for compatibility tests it won't matter, thanks for the explanation.

Reply:
Yeah, for larger captures, GCS makes more sense. I guess for this compatability tests, it is fine to have them in Github.